emr-dynamodb-hadoop/src/main/java/org/apache/hadoop/dynamodb/write/WriteIopsCalculator.java

```diff
@@ -13,6 +13,7 @@
 
 package org.apache.hadoop.dynamodb.write;
 
+import com.amazonaws.services.dynamodbv2.model.TableDescription;
```

Comment:
if we're removing `getThroughput()`, this line is used, please consider to remove it